src/logging.js

```diff
@@ -2,6 +2,12 @@ Room.prototype.log = function(...messages) {
   console.log(`${Game.time} ${this.name.rpad(' ', 27)} ${messages.join(' ')}`);
 };
 
+Room.prototype.debugLog = function(type, ...messages) {
+  if (config.debug[type]) {
```

Comment:
maybe add a `test flag` for circleci.com, like 
````
  if (config.debug[type] || config.debug.circleci) {
```
for loging everything?

Reply:
Actually, I'm introducing this to reduce the noise to make exceptions or unhandled cases more prominent.

We could introduce a `config_local.js` for circleCi, for now I would like to reduce the log output.